rll/metadata.rb

```diff
@@ -58,3 +51,9 @@
   :default => "cred:AWS_ACCESS_KEY_ID",
   :recipes => ["rll::test-script"]
 
+attribute   "DISABLE_AUTO_UPGRADE",
+  :display_name => "Disables auto upgrade if previously enabled",
```

Comment:
Initially running `setup_automatic_upgrade` enables auto upgrade...this is what is meant by 'previously enabled'.  An extra action should be done to disable it, which is setting this attribute to 'true'.  It would be easy to flip the logic and change it to `ENABLE_AUTO_UPGRADE` attribute and default it to 'true', and if they with to disable auto upgrade, set that to 'false' - is that what is being suggested?